internal/version/version.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Lekko Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+)
+
+var (
+	//go:embed version.txt
+	version    string
+	Version    = readVersion()
+	SDKVersion = fmt.Sprintf("go-%s", Version)
```

Comment:
Maybe OOTL, what's the motivation for having 2 different types of versions (where the only difference is a `go-` prefix)?

Reply:
one is for unit tests (to make sure its semver), and the other is prefixed with `go-` for sending back to the backend. I just changed to only make the latter public